code/scripting/api/objs/hudconfig.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "hud/hudconfig.h"
+#include "scripting/ade_api.h"
+
+namespace scripting {
+namespace api {
+
+struct gauge_config_h {
```

Comment:
please doxy this struct and its members

Reply:
There is not a single handle struct in the lua api that I could find that is doxy'd. I'll do it if you insist, but this will be the only one.

Reply:
nah, it should be fine for now.